csv2rdf/index.html

```diff
@@ -893,13 +778,15 @@
                   the <code>name</code> property specified for each
                   <a href="http://www.w3.org/TR/tabular-data-model/#dfn-column">column</a>. 
                   During template expansion, the <em>variables</em> evaluate to the 
-                  <a href="http://www.w3.org/TR/tabular-data-model/#dfn-string-value">cell value</a>
+                  <a href="http://w3c.github.io/csvw/metadata/#dfn-cell-value">cell value</a>
                   within the 
                   <a href="http://www.w3.org/TR/tabular-data-model/#dfn-row">row</a>
                   being processed that is associated with the named 
                   <a href="http://www.w3.org/TR/tabular-data-model/#dfn-column">column</a>.</p>
                 <p>The <em>variable</em> <code>_row</code> evaluates to the number of the row being
```

Comment:
Clarification... Do you mean that the value of ``_value`` is the raw cell value, as opposed to the one that may have been transformed to match the datatype definition?

Ivan

> On 04 Feb 2015, at 13:04 , Jeni Tennison <notifications@github.com> wrote:
> 
> In csv2rdf/index.html:
> 
> > @@ -893,13 +778,15 @@
> >                    the <code>name</code> property specified for each
> >                    <a href="http://www.w3.org/TR/tabular-data-model/#dfn-column">column</a>.
> >                    During template expansion, the <em>variables</em> evaluate to the
> > -                  <a href="http://www.w3.org/TR/tabular-data-model/#dfn-string-value">cell value</a>
> > +                  <a href="http://w3c.github.io/csvw/metadata/#dfn-cell-value">cell value</a>
> >                    within the
> >                    <a href="http://www.w3.org/TR/tabular-data-model/#dfn-row">row</a>
> >                    being processed that is associated with the named
> >                    <a href="http://www.w3.org/TR/tabular-data-model/#dfn-column">column</a>.</p>
> >                  <p>The <em>variable</em> <code>_row</code> evaluates to the number of the row being
> 
> I think we need to add another variable to support the valueUrl property, ie _value evaluating to the cell value of the cell being processed.
> 
> —
> Reply to this email directly or view it on GitHub.
> 


----
Ivan Herman, W3C
Digital Publishing Activity Lead
Home: http://www.w3.org/People/Ivan/
mobile: +31-641044153
ORCID ID: http://orcid.org/0000-0003-0782-2704